macros/sql/star.sql

```diff
@@ -12,7 +12,7 @@
     {%- endif -%}
 
     {%- set include_cols = [] %}
-    {%- set cols = adapter.get_columns_in_table(schema_name, table_name) -%}
+    {%- set cols = adapter.get_columns_in_relation(this) -%}
```

Comment:
If `from` is a Relation, then we can use that directly. I think we should change the contract of this macro to _require_ that a Relation is provided. It's going to be really hard for us to marshal a `schema.table` string into a Relation in an intelligent way. @jtcohen6 agree?

Reply:
Agree. The only way I could imagine accepting schema + identifier args is if we called the api get (create) relation function as well, which seems a worse pattern than enforcing that the argument be a relation.